index.js

```diff
@@ -69,7 +69,9 @@ function polylabel(polygon, precision, debug) {
         console.log('best distance: ' + bestCell.d);
     }
 
-    return [bestCell.x, bestCell.y];
+    var result = [bestCell.x, bestCell.y];
+    result.distance = bestCell.d;
```

Comment:
I've seen properties on arrays in D3, for example `data.columns` in [dsv.parse](https://github.com/d3/d3-dsv#dsv_parse)